sonarqube/resource_sonarqube_alm_github.go

```diff
@@ -0,0 +1,188 @@
+package sonarqube
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// GetAlmGithub for unmarshalling response body from alm list definitions. With only github populated
+type GetAlmGithub struct {
+	Github []struct {
+		Key      string `json:"key"`
+		URL      string `json:"url"`
+		AppID    string `json:"appId"`
+		ClientID string `json:"clientId"`
+	} `json:"github"`
+}
+
+// Returns the resource represented by this file.
+func resourceSonarqubeAlmGithub() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSonarqubeAlmGithubCreate,
+		Read:   resourceSonarqubeAlmGithubRead,
+		Update: resourceSonarqubeAlmGithubUpdate,
+		Delete: resourceSonarqubeAlmGithubDelete,
+
+		// Define the fields of this schema.
+		Schema: map[string]*schema.Schema{
+			"appid": {
```

Comment:
@jdamata Im aware of these attributes not following the style guide for terraform ( which i believe is snake case) and i will solve this tommorow after i got some friday night sleep 😄

Reply:
sounds good :)

Reply:
@jdamata Its done ,i also added imports. Do note that the tests are bound to 8.9 now, While LTS is 9.8 i think because sonarqube changed it yesterday